### ISSUE TITLE
IBM i Output view stops showing compile list

### ISSUE THREAD

**Issue by richardm90** [halcyon-tech/vscode-ibmi#928]
I've found that after I've been using VS Code for a while, maybe a couple of hours, the compile listings stop showing in the IBM i Output view. It happens consistently during the working day though I'm not sure on the exact timing or circumstance. To resolve this I generally reopen VS Code although I have had limited success with clearing output in both the IBM i Output and Code for i views but this generally only works for the next one or two compiles.

I have noticed occasionally that the Code for i view appears to be looping through some tasks within me doing anything.

It feels like to me something is definitely "filling up" after a certain amount of time and reopening VS Code clears it.

* QCCSID: 1146
* Features:
   * git: true
   * grep: true
   * tn5250: false
   * setccsid: true
   * GENCMDXML.PGM: false
   * QZDFMDB2.PGM: true
   * iconv: true
   * attr: true
* SQL enabled: true
* Source dates enabled: true

Variants
```json
{
  "american": "%23@$",
  "local": "%23@£"
}
```

Errors:
```json
[
  {
    "command": "system \"DLTOBJ OBJ(RICHARD/O_*) OBJTYPE(*FILE)\"",
    "code": 255,
    "stderr": "CPF2125: No objects deleted.",
    "cwd": "/home/richardm"
  },
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPTOIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Object QCPTOIMPF in library QSYS not found.",
    "cwd": "/home/richardm"
  },
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPFRMIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Object QCPFRMIMPF in library QSYS not found.",
    "cwd": "/home/richardm"
  }
]
```

**Comment by richardm90** [comment]
https://user-images.githubusercontent.com/7809440/198707907-05025022-dbd1-43a5-a4b5-8cbe8df6477b.mp4

This video shows an example of the Code for i view looping through tasks but this isn't always the case when the compile listing stops showing.

**Comment by worksofliam** [comment]
Looks like there is two issues here:

1. Code for IBM i output likely growing too large.
2. Commands being executed repeatedly for no reason, though only happens after a couple of hours of VS Code being active/connected.

**Comment by worksofliam** [comment]
Regarding point 2: the code it's running is creating and and fetching from an alias. This only happens through the QSYS File System provider when source dates are enabled, which VS Code calls internally. That could potentially be a bug in VS Code internals.

**Comment by worksofliam** [comment]
Regarding points 1: I've added something internally to clear it out every so often. This will be much nicer. I was under the assumption VS Code would clean it up automatically. Silly me for thinking that.

Regarding point 2: Yes, it's a bug with VS Code and preview mode. I was able to recreate it and I have seen something like this before. Disable this feature for now (in the settings - photo below) and restart VS Code. That should solve the problem. I might add something to disable it by default when a user connects to a system.
